output/cloud/expv2/flush.go

```diff
@@ -131,6 +157,20 @@ func (msb *metricSetBuilder) addTimeBucket(bucket timeBucket) {
 	}
 }
 
+func (msb *metricSetBuilder) recordDiscardedLabels(labels map[string]struct{}) {
```

Comment:
```suggestion
func (msb *metricSetBuilder) recordDiscardedTags(keys []string) {
```

Reply:
I'd prefer to keep the `lables` naming here since, in the proto we call them labels.